src/prx/rinex_nav/evaluate.py

```diff
@@ -24,9 +24,12 @@ def cached_parse(rinex_file: Path, file_hash: str):
     @helpers.disk_cache.cache
     def cached_load(rinex_file: Path, file_hash: str):
         ds = cached_parse(rinex_file, file_hash)
-        ds.attrs["utc_gpst_leap_seconds"] = (
-            helpers.get_gpst_utc_leap_seconds_from_rinex_header(rinex_file)
-        )
+        try:
+            ds.attrs["utc_gpst_leap_seconds"] = (
+                    helpers.get_gpst_utc_leap_seconds_from_rinex_header(rinex_file)
+            )
+        except:
+            ds.attrs["utc_gpst_leap_seconds"] = 18
```

Comment:
This solution is not future proof. I have just coded a `leap seconds` function, that I will push to your branch.